server/startup/migrations/v219.js

```diff
@@ -0,0 +1,35 @@
+import { Migrations } from '../../../app/migrations/server';
```

Comment:
You're not importing this file anywhere(`./index.js`).

Reply:
I've imported it now, and its working